lib/terrafying/components/vpc.rb

```diff
@@ -92,6 +92,10 @@ def create(name, raw_cidr, options = {})
           ssh_group: DEFAULT_SSH_GROUP
         }.merge(options)
 
+        if !options[:provider].nil?
+          @opts_provider = options[:provider]
+        end
```

Comment:
This is fine, two things for interest though:

1. You can use `unless` if you want to invert an if. For example:
```
unless options[:provider].nil?
  @opts_provider = options[:provider]
end
```
or shorter
```
@opts_provider = options[:provider] unless options[:provider].nil?
```

2. You can skip the conditional entirely since `@opts_provider` is either `undefined` or `nil` at this point already and assigning `nil` to it won't change the logic later as both are falsy.